Gemfile.lock

```diff
@@ -405,69 +408,70 @@ GEM
       terminal-table (>= 1.4.0)
       thor (>= 0.16.0)
     jmespath (1.6.2)
-    json (2.6.3)
-    jsonpath (1.1.2)
+    json (2.7.1)
+    jsonpath (1.1.5)
       multi_json
     jwt (2.7.1)
     kubeclient (4.11.0)
       http (>= 3.0, < 6.0)
       jsonpath (~> 1.0)
       recursive-open-struct (~> 1.1, >= 1.1.1)
       rest-client (~> 2.0)
-    large_object_store (1.3.4)
-    llhttp-ffi (0.4.0)
+    large_object_store (1.7.0)
+      zstd-ruby (~> 1.5.5)
+    llhttp-ffi (0.5.0)
       ffi-compiler (~> 1.0)
       rake (~> 13.0)
-    lograge (0.12.0)
+    lograge (0.14.0)
       actionpack (>= 4)
       activesupport (>= 4)
       railties (>= 4)
       request_store (~> 1.0)
     logstash-event (1.2.02)
-    loofah (2.21.4)
+    loofah (2.22.0)
       crass (~> 1.0.2)
       nokogiri (>= 1.12.0)
     mail (2.7.1)
       mini_mime (>= 0.1.1)
     marcel (1.0.2)
     marco-polo (2.0.1)
       railties (>= 6)
-    maxitest (3.3.0)
-      minitest (>= 5.0.0, < 5.12.0)
-    metaclass (0.0.4)
-    method_source (0.8.2)
-    mime-types (3.4.1)
+    maxitest (3.7.0)
+      minitest (>= 5.0.0, < 5.15.0)
+    method_source (0.9.2)
+    mime-types (3.5.2)
```

Comment:
### [mime-types](https://github.com/mime-types/ruby-mime-types/)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 3.4.1 → 3.5.2

[_[change-log](https://github.com/mime-types/ruby-mime-types/blob/master/History.md), [source-code](https://github.com/mime-types/ruby-mime-types/), [gem-diff](https://my.diffend.io/gems/mime-types/3.4.1/3.5.2)_]





<details>
<summary>Commits</summary>

A change of **20** commits. See the full changes on [the compare page](https://github.com/mime-types/ruby-mime-types/compare/v3.4.1...v3.5.2).

These are the first 10 commits:
- (3c41051) [Add Ruby 3.1 to the CI matrix](https://github.com/mime-types/ruby-mime-types/commit/3c41051f519d66e1a0d0f0438a4e34cd053e93f0)
- (446d7d7) [Define MIME::Type#hash](https://github.com/mime-types/ruby-mime-types/commit/446d7d78b332b764d7af6ad1a38efea2a67c4416)
- (071107b) [Merge pull request #167 from ajvondrak/define-mime-type-hash](https://github.com/mime-types/ruby-mime-types/commit/071107b627f606a04b20e6fe93e7d19adcb2277a)
- (da5d341) [Update changelog for PR #167](https://github.com/mime-types/ruby-mime-types/commit/da5d341be769f2a9b234cf986a75ca98de64f9af)
- (27293f5) [Merge pull request #165 from mishina2228/add-ruby31-to-ci-ma…](https://github.com/mime-types/ruby-mime-types/commit/27293f53a325c00144df41e38a29e8ec1e2dea7a)
- (44a3787) [Clean up Travis CI](https://github.com/mime-types/ruby-mime-types/commit/44a3787f36371c9a51cc4c7b521eb9c5d87d9221)
- (3ba14a9) [Merge pull request #164 from mishina2228/clean-up-travis](https://github.com/mime-types/ruby-mime-types/commit/3ba14a9a96d5156dc5f670239b7cf42171b86528)
- (b1b8e4e) [Add Ruby 3.2 to the CI matrix](https://github.com/mime-types/ruby-mime-types/commit/b1b8e4e674786b1e76dd77c798b5414ff31f55e2)
- (6910344) [Add fail-fast: false](https://github.com/mime-types/ruby-mime-types/commit/69103440cb335581023bdcba7b26cefadbd3cb25)
- (0bf7805) [Merge pull request #169 from petergoldstein/feature/add_ruby…](https://github.com/mime-types/ruby-mime-types/commit/0bf7805068a6b8e53abbe5ac7c09aedda63ee54f)


</details>